requirements.txt

```diff
@@ -10,4 +10,5 @@ mysqlclient==2.1.1
 arrow~=1.3.0
 python-graphql-client~=0.4.3
 djangorestframework>=3.14
-bugsnag
\ No newline at end of file
+bugsnag
+netifaces
```

Comment:
Can we sort this whitespace? @theahthodesen

Reply:
fixed